SecretSanta.sql

```diff
@@ -3,27 +3,27 @@ create table Users (
 	name text,
 	username text,
 	password text,
-    `time` timestamp
+    	`time` timestamp
 );
 
 create table Rooms (
 	room_id int primary key auto_increment,
 	admin_user_id int,
 	title text,
 	table_status boolean,
-    room_url integer,
-    room_password text,
-    `time` timestamp,
-    foreign key (admin_user_id) references Users(user_id)
+    	room_url integer,
+   	room_password text,
+    	`time` timestamp,
+    	foreign key (admin_user_id) references Users(user_id)
 );
 
 create table RoomUsers (
 	room_user_id integer,
```

Comment:
Room user ID should be a primary key and should auto increment, don't you think?